api/src/api/notebooks.ts

```diff
@@ -58,19 +65,26 @@ import {
 import * as Exceptions from '../exceptions';
 import {requireAuthenticationAPI} from '../middleware';
 
+// See https://github.com/davidbanham/express-async-errors - this patches
+// express to handle async errors without hanging or needing an explicit try
+// catch block
+require('express-async-errors');
```

Comment:
Why is this 'require' rather than 'import'?

Reply:
Just using their docs example - but I'll see if the behaviour works with import.

Reply:
Just tested and as discussed there is no typing on the patch module - require is a cleaner way to indicate/hide this.